make/modules/java.desktop/lib/ClientLibraries.gmk

```diff
@@ -281,51 +281,51 @@ endif
 ################################################################################
 
 ifeq ($(USE_EXTERNAL_HARFBUZZ), true)
-   LIBFONTMANAGER_EXTRA_SRC =
-   LIBFONTMANAGER_LIBS += $(HARFBUZZ_LIBS)
-   LIBFONTMANAGER_CFLAGS += $(HARFBUZZ_CFLAGS)
+  LIBFONTMANAGER_EXTRA_SRC =
```

Comment:
I think this 3space -> 2space indentation change shouldn't be part of this PR, esp. since 3space is
used in other parts of this file.

Reply:
Really? That is sooo broken. :( But fair enough, I'll do a separate PR on the broken indentation.